### PR TITLE
Pull to refresh

### PR DESCRIPTION
**Pull request by oskarmt**
Fix pull to refresh.

### CODE REVIEW

File: Football/Team.swift [hunk 1]
```diff
@@ -18,7 +18,8 @@ class Team {
     var teamPos : Int
     var leagueCaption = "Premier League 2015/16"
     
-    init(teamName: String, teamPoint: Int, teamGame: Int, teamWin: Int, teamLosse : Int, teamDraw : Int, teamPos : Int ){
+    init(teamName: String, teamPoint: Int, teamGame: Int, teamWin: Int,
```
Comment (julia-kurilchik) [inline Football/Team.swift]: rename teamGame to teamGamesCount, or something like this

File: Football/FirstTableViewController.swift [hunk 2]
```diff
@@ -31,26 +33,27 @@ class FirstTableViewController: UITableViewController {
...
-                print("refreshing")
-                sleep(2)
-                dispatch_async(dispatch_get_main_queue()) {
-                    refreshEnd(0)
-                }
+    func refreshBegin(newtext:String, refreshEnd:(Int) -> ()) {
+        dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0)) {
+            print("refreshing")
+            self.loadData()
+            sleep(2)
```
Comment (julia-kurilchik) [inline Football/FirstTableViewController.swift]: why do we need it?

Comment (oskarmt) [inline Football/FirstTableViewController.swift]: It is delay for show user what refreshing was activated.

**Comment by julia-kurilchik** [comment]
:+1:

**Pull request merged**